xbmc/GUIInfoManager.cpp

```diff
@@ -332,6 +337,33 @@ const infomap integer_bools[] =  {{ "isequal",          INTEGER_IS_EQUAL },
 ///     Returns true if pvr channel preview is active (used channel tag different
 ///     from played tag)
 ///   }
+///   \table_row3{   <b>`Player.TempoEnabled`</b>,
+///                  \anchor Player_TempoEnabled
+///                  _boolean_,
+///     Returns true if player supports tempo (i.e. speed up/down normal playback speed)
+///   }
+///   \table_row3{   <b>`Player.IsTempo`</b>,
+///                  \anchor Player_IsTempo
+///                  _boolean_,
+///     Returns true if player has tempo (i.e. is playing with a playback speed higher or lower than normal playback speed)
+///   }
+///   \table_row3{   <b>`Player.PlaySpeed`</b>,
+///                  \anchor Player_PlaySpeed
+///                  _string_,
+///     Returns the player playback speed (tempo) with the format %1.2f (1.00 means normal playback speed)
+///   }
+///   \table_row3{   <b>`Player.HasResolutions`</b>,
+///                  \anchor Player_HasResolutions
+///                  _boolean_,
+///     Returns true if the player is allowed to switch resolution and refresh rate (i.e. if whitelist modes are configured in Kodi's System/Display settings)
+///   }
+///   \table_row3{   <b>`Player.HasPrograms`</b>,
+///                  \anchor Player_HasPrograms
+///                  _boolean_,
+///     Returns true if the media file being played has programs\, i.e. groups of streams. 
```

Comment:
inconsistencies were seen between `ffmpeg` and `inputstream.adaptive` in the way both interpret available streams. Might be important to clarify.